docs/advanced-topics/best-practices/best-practices-secure-fair-workflows.rst

```diff
@@ -0,0 +1,174 @@
+Best Practices for Secure and FAIR tools and workflows using Dockstore
+========================
+
+This comprehensive document contains best practices for developing secure tools or workflows that also exemplify the `FAIR (Findable, Accessible, Interoperable, Reusable) guiding principles <https://www.go-fair.org/fair-principles/>`_. 
+
+Version Control Best Practices
+-----------------------------------
+
+- Host your source code, workflow descriptor file, and Dockerfile in a git repository. Dockstore currently supports GitHub, BitBucket, and GitLab. We recommend GitHub because the `GitHub App integrates easily with Dockstore <https://docs.dockstore.org/en/develop/getting-started/github-apps/github-apps-landing-page.html?highlight=github>`_. If you are new to using version control, you can start with these introductory documents:
```

Comment:
It's best to use a relative path our own doc. That way if we do a different release than develop it would work. Only commenting here, but there are several instances of it in the doc.